### PR TITLE
fix: add possibility to define dicts w/o values in checks and report

### PR DESCRIPTION
**Pull request by FoSix**
## Description

Add flexibility to define a check or report state with `key: None` format, which equals YAML's `key:`

## Motivation and Context

see description in #100 

## How Has This Been Tested?

Existing python code was checked (Examples)

## Types of changes

- Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: coverage.xml [hunk 1]
```diff
@@ -0,0 +1,827 @@
+<?xml version="1.0" ?>
```
Comment (alperenkose) [inline coverage.xml:1]: We shouldn't commit this file.

**Review by alperenkose — CHANGES_REQUESTED** [review]
Looks good to me, just the coverage.xml need to be removed.

File: panos_upgrade_assurance/check_firewall.py [hunk 2]
```diff
@@ -949,10 +949,10 @@ def run_readiness_checks(
         for check in checks_list:
             if isinstance(check, dict):
                 check_type, check_config = next(iter(check.items()))
-                # check_result = self._check_method_mapping[check_type](check_config)
+                if check_config is None:
```
Comment (alperenkose) [inline panos_upgrade_assurance/check_firewall.py:952]: I think we should include dicts with null value in the unittests.

Comment (FoSix) [reply panos_upgrade_assurance/check_firewall.py:952]: covered in [this commit](https://github.com/PaloAltoNetworks/pan-os-upgrade-assurance/pull/101/commits/706ff8d2ad34662f89b8f2e42f9d3bb90bb7c2bd)

**Pull request merged**